app/helpers/get_involved_helper.rb

```diff
@@ -0,0 +1,31 @@
+module GetInvolvedHelper
+  def date_microformat(attribute_name)
+      attribute_name.to_date.strftime("%d %B %Y")
+  end
+
+  # Gets the link to the search page for all consultations
+  def get_consultations_link(filters = %w[open_consultations closed_consultations])
+    "search/policy-papers-and-consultations?#{filters.to_query("content_store_document_type")}"
+  end
+
+  private
+
+  def render_datetime_microformat(object, method, &block)
```

Comment:
Is this actually used anywhere? 🤔

Reply:
This is a remnant from the change in Line 3. Previously it did all of this extra to format the time but I optimised it in Line 3 and didn't actually remove the original function. Good catch!